### PR TITLE
[4.4] Backport new timeout config options [ADR 006]

### PR DESCRIPTION
**Pull request by robsdedude**
Backport of https://github.com/neo4j/neo4j-python-driver/pull/745

### CODE REVIEW

File: neo4j/conf.py [hunk 1]
```diff
@@ -256,6 +263,12 @@ class WorkspaceConfig(Config):
     """ WorkSpace configuration.
     """
 
+    #: Session Connection Timeout
+    session_connection_timeout = 120.0  # seconds
```
Comment (bigmontz) [inline neo4j/conf.py]: Shouldn't it be inf?

**Review by bigmontz — CHANGES_REQUESTED** [review]
the default `session_connection_timeout` seems wrong.

File: neo4j/conf.py (hunk 1 above)
Comment (robsdedude) [reply neo4j/conf.py]: It 100%... nah inf% should be! :open_mouth:

**Review by bigmontz — APPROVED** [review]
⏳

**Pull request merged**